locksmith/src/websub/helpers/renewKey.ts

```diff
@@ -0,0 +1,130 @@
+import { ethers, Wallet, Contract, constants } from 'ethers'
+import { networks } from '@unlock-protocol/networks'
+import { NetworkConfig } from '@unlock-protocol/types'
+import { KeyRenewal } from '../../models'
+import { purchaserCredentials } from '../../../config/config'
+
+interface RenewKeyParams {
+  keyId: number
+  lockAddress: string
+  network: number
+}
+
+const abi = [
```

Comment:
Let's not use the `abi` here but unlockJS which is the ability to return the lock contract at a specific address so that we don't have to maintain ABIs in multiple places.